logging/log_storage/cluster-logging-loki.adoc

```diff
@@ -18,8 +18,6 @@ include::modules/logging-loki-restart-hardening.adoc[leveloffset=+1]
 
 include::modules/logging-loki-reliability-hardening.adoc[leveloffset=+1]
 
-include::modules/logging-loki-pod-placement.adoc[leveloffset=+1]
```

Comment:
AFAICS you need to split this module into two modules one for the node selectors page and one for the tolerations page. If possible it would be nice to rename the titles from `Loki pod placement` to:
- For NodeSelectors: "Configuring scheduling to control LokiStack pod placement"
- For Tolerations: "Using tolerations to control LokiStack pod placement"

Reply:
I am aware of that, but for now I have included it in both places instead. We can open a follow up issue to split this module, but it would add quite a bit of work that for me is too far outside the scope of this issue to complete right now.